libbeat/scripts/Makefile

```diff
@@ -240,6 +240,7 @@ update: python-env
 	cp ${ES_BEATS}/libbeat/docs/version.asciidoc docs/version.asciidoc
 
 	# Generate index-pattern
+	mkdir -p $(PWD)/etc/kibana/index-pattern
```

Comment:
@ruflin So far we have only one index-pattern per Beat, so I added into the Makefile to delete the index template before generating a new one.